public/stylesheets/style.css

```diff
@@ -1,8 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
-  padding: 50px;
-  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  /* border: 1px solid black; */
+  /* background-color: #f5f5f5; */
+  font-family: "Roboto", sans-serif;
+  margin: auto;
+  margin-top: 60px;
+  height: 500px;
+  width: 400px;
+}
+
+header {
+  background-color: #0d213a;
+  text-align: center;
+  padding-top: 20px;
+  padding-bottom: 10px;
+}
+
+h1 {
+  color: #fff;
+  font-size: 20px;
+  margin-top: 0;
+}
+
+p {
+  color: #fff;
+  font-size: 13px;
 }
 
-a {
-  color: #00B7FF;
+form {
+  background-color: #fff;
+  box-shadow: 0 2.8px 2.2px rgba(0, 0, 0, 0.034),
+    0 6.7px 5.3px rgba(0, 0, 0, 0.048), 0 12.5px 10px rgba(0, 0, 0, 0.06),
+    0 22.3px 17.9px rgba(0, 0, 0, 0.072), 0 41.8px 33.4px rgba(0, 0, 0, 0.086),
+    0 100px 80px rgba(0, 0, 0, 0.12);
+  padding: 30px 20px;
 }
+
+label {
+  color: #7379a3;
+  display: block;
+  font-size: 15px;
+  margin-top: 30px;
+  margin-bottom: 10px;
+}
+
+input[type="text"] {
+  border: 1px solid #ecf2fa;
+  border-radius: 5px;
+  color: #7379a3;
+  padding: 10px 0 10px 10px;
+  width: 100%;
+}
+
+input[type="submit"] {
```

Comment:
Dont use elements as selectors when they are not scoped. Use either a class or ID or scope it